packages/twenty-server/src/engine/api/graphql/workspace-query-runner/factories/query-runner-args.factory.ts

```diff
@@ -210,30 +206,12 @@ export class QueryRunnerArgsFactory {
             'Rich text is not supported, please use RICH_TEXT_V2 instead',
           );
         case FieldMetadataType.RICH_TEXT_V2: {
-          const richTextV2Value = richTextV2ValueSchema.parse(value);
-
-          const serverBlockNoteEditor = ServerBlockNoteEditor.create();
-
-          const convertedMarkdown = richTextV2Value.blocknote
-            ? await serverBlockNoteEditor.blocksToMarkdownLossy(
-                JSON.parse(richTextV2Value.blocknote),
-              )
-            : null;
-
-          const convertedBlocknote = richTextV2Value.markdown
-            ? JSON.stringify(
-                await serverBlockNoteEditor.tryParseMarkdownToBlocks(
-                  richTextV2Value.markdown,
-                ),
-              )
-            : null;
-
-          const valueInBothFormats: RichTextV2Metadata = {
-            markdown: richTextV2Value.markdown || convertedMarkdown,
-            blocknote: richTextV2Value.blocknote || convertedBlocknote,
-          };
-
-          return [key, valueInBothFormats];
+          const overriddenValue = await overrideFieldValue(
```

Comment:
this is very close to the arg-setter logic we have for graphql already.

we use it to set the convert position: last to its actual value for example. This seems to be a duplicate pattern of it

Long term, I feel the vision should be to add this logic to the ORM layer